substrate/frame/executive/src/lib.rs

```diff
@@ -115,7 +122,23 @@
 //!     }
 //! }
 //!
-//! pub type Executive = executive::Executive<Runtime, Block, Context, Runtime, AllPalletsWithSystem, CustomOnRuntimeUpgrade>;
+//! struct CustomTryState;
+//!
+//! #[cfg(feature = "try-runtime")]
+//! impl frame_support::traits::IdentifiableTryStateLogic<BlockNumber> for CustomTryState {
+//!
+//! 	fn try_state(_: BlockNumber) -> Result<(), sp_runtime::TryRuntimeError> {
```

Comment:
Mhh but the trait `IdentifiableTryStateLogic` is also implemented for `AllPalletsWithSystem`, so having a `before_all_pallets` method would not be ideal naming, I think. Also, this would be implemented by each pallet, so for a pallet to implement a function `before_all_pallets` can get quite confusing. 

I'll shortly update the PR to add more context, but I also tried to keep the changeset to the minimum without changing too many things from how things work today.

Reply:
Ah, i mixed it up with runtime-wide post migration check. nvm